src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -120,7 +120,11 @@ zocl_load_bitstream(struct drm_zocl_dev *zdev, char *buffer, int length)
 		data[i+2] = temp;
 	}
 
-	return zocl_load_partial(zdev, data, bit_header.BitstreamLength);
+	/* On pr platofrm load partial bitstream and on Flat platform load full bitstream */
+	if (zdev->pr_isolation_addr)
+		return zocl_load_partial(zdev, data, bit_header.BitstreamLength);
+	else
+		return zocl_fpga_mgr_load(zdev, buffer, length, 0); //0 is for full btstream
```

Comment:
besides the typo, can we please change this to
else {
        /* 0 is for full bitstream */
        return ...
}

Reply:
fixed